web/src/features/menu/context/Item.tsx

```diff
@@ -66,17 +67,20 @@ const Item: React.FC<{
                   }}
                 />
               )}
-              <Box>
-                <Box paddingBottom={button.description ? 1 : 0}>
+              <Box w="100%">
+                <Box>
                   <Text w="100%" fontWeight="medium" color={button.disabled ? '#718096' : undefined}>
                     {button.title ? button.title : buttonKey}
                   </Text>
                 </Box>
-                {button.description && (
-                  <Box paddingBottom={1} color={button.disabled ? '#718096' : undefined}>
+                { button.description && (
+                  <Box paddingBottom={1} color={button.disabled ? '#718096' : undefined} fontWeight="light">
```

Comment:
Not a fan of the lighter look on the font at all, again, this is something people will be able to change more easily when everything is moved over to scss 🤷